config/all-in-one/templates/eventmanager-app.yaml

```diff
@@ -0,0 +1,89 @@
+{{- if .Values.cp4waiops.eventManager.enabled }}
+apiVersion: argoproj.io/v1alpha1
+kind: ApplicationSet
+metadata:
+  name: eventmanager-app
+spec:
+  generators:
+  {{- if .Values.argocd.allowLocalDeploy }}
+  - clusters: {}
+  {{- else }}
+  - clusters:
+      selector:
+        matchLabels:
+          argocd.argoproj.io/secret-type: cluster
+  {{- end }}
+  template:
+    metadata:
+      name: '{{ `{{name}}-eventmanager` }}'
+      annotations:
+        argocd.argoproj.io/sync-wave: "200"
+    spec:
+      destination:
+        namespace: '{{ .Values.cp4waiops.eventManager.namespace }}'
+        server: '{{ `{{server}}` }}'
+      project: default
+      source:
+        path: {{ template "eventmanager.configPath" . }}
+        repoURL: {{ default "https://github.com/IBM/cp4waiops-gitops" .Values.cp4waiops.repoURL }}
+        targetRevision: {{ default "HEAD" .Values.cp4waiops.targetRevision }}
+        helm:
+          valueFiles:
+            - values.yaml
```

Comment:
It is not needed because that's Argo CD's default behavior.

Reply:
Removed